tests/ut_touchscreen/ut_touchscreen.cpp

```diff
@@ -112,12 +112,6 @@ void Ut_TouchScreen::testTouchBlocking()
     touchBlockingSpy.wait();
     QVERIFY(!touchScreen->touchBlocked());
 
-    event = QMouseEvent(QEvent::MouseButtonPress, QPointF(), Qt::NoButton, 0, 0);
-    QCOMPARE(touchScreen->eventFilter(0, &event), true);
-
-    event = QMouseEvent(QEvent::MouseMove, QPointF(), Qt::NoButton, 0, 0);
```

Comment:
Below this seems to test that touchupdate and touchend without touchbegin first are filtered out. If mouse would be handled in similar way, this latter should still be valid test?

Reply:
Nope.

Previously: everything was filtered out until TouchBegin is seen i.e. the first compare below that expects false return value.

Currently: Filtering stops when TouchBegin or MouseButtonPress or MouseMove is seen -> Test code expecting previous behavior fails when MouseButtonPress returns false as it should.

The fix is: Have the original test block deal  with everything that ought to be filtered away and that  TouchBegin still ends filtering. And then add two new minimalistic blocks that ensure that also  MouseButtonPress and MouseMove events stop filtering.